src/MIDI.hpp

```diff
@@ -118,6 +119,45 @@ void MidiInterface<SerialPort, Settings>::begin(Channel inChannel)
  @{
  */
 
+/*! \brief Send a MIDI message.
+\param inMessage    The message
+ 
+ This method is used when you want to send a Message that has not been constructed
+ by the library, but by an external source.
+ This method does *not* check against any of the constraints.
+ Typically this function is use by MIDI Bridges taking MIDI messages and passing
+ them thru.
+ */
+template<class Transport, class Settings, class Platform>
+void MidiInterface<Transport, Settings, Platform>::send(MidiMessage inMessage)
```

Comment:
Maybe we should pass the message as a const ref, to avoid copies (better performance and memory usage): `::send(const MidiMessage& inMessage)`

Reply:
good catch!